Prose/ProseLib/Sources/ConversationFeature/Chat/Chat.swift

```diff
@@ -126,7 +126,7 @@ struct ChatView: NSViewRepresentable {
     webView.loadFileURL(Files.messagingHtml.url, allowingReadAccessTo: Files.messagingHtml.url)
 
     context.coordinator.ffi = FFI { [weak webView] jsString, completion in
-      jsLogger.trace("\(jsString, privacy: .public)")
+      jsLogger.debug("\(jsString)")
```

Comment:
Why `debug` now? `trace` was good, it's not the most useful log, so we'd better not keep it over a useful `debug` log. From [the documentation](https://developer.apple.com/documentation/os/logger/3551624-trace), `trace` is functionally equivalent to `debug`:

> This method is functionally equivilent to the [debug(_:)](https://developer.apple.com/documentation/os/logger/3551615-debug) method.

I searched in the documentation for the article in which they detail log retention priorities, but I can't seem to find it.

Reply:
For no reason, that's just me being confused 🤦‍♂️

Reply:
I'm confused now too if you say it's not persisted but obfuscated… what's the point if it always runs on dev environment?

Reply:
I really wish I find the Apple article about log retention again, maybe in the wayback machine…

Reply:
Ok, a few things got mixed up here. First of all retention is documented in the WWDC video and article someone kindly shared, as I've mentioned in my comment [here](https://github.com/prose-im/prose-app-macos/pull/105#discussion_r943276307).

`trace` and `debug` messages are not obscured when debugging and streaming to Console.app as I've incorrectly assumed and tried to clarify.

This only happens when `OS_ACTIVITY_DT_MODE` is set to `NO`.

More info on that environment variable [here](https://developer.apple.com/forums/thread/705810) and [here](https://codehunter.cc/a/xcode/hide-strange-unwanted-xcode-logs).